packages/dds/matrix/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     },
     "include": [
         "src/**/*"
-    ],
+, "src/test/memory/.mocharc.js"    ],
```

Comment:
Is this line necessary? With the change in `src/test/tsconfig.json` I'd expect this one not to be required.